controllers/userControllers.js

```diff
@@ -0,0 +1,111 @@
+const User = require("../models/User");
+const bcryptjs = require("bcryptjs");
+
+/**
+ * things to think about
+ * how we grab users : through req.body or req.params
+ *
+ */
+
+exports.getUsers = async (req, res) => {
+  let users = await User.find();
+  res.json(users);
+};
```

Comment:
when do we need to get ALL users at once? or is it just for checking reasons and will be deleted later on?

Reply:
yeah we will only use this for testing users and not deploy when we use our own accounts